frontend/helpers/risks.ts

```diff
@@ -123,14 +124,12 @@ export const getRiskLikelihoodName = (likelihood: number) => {
 	return (filteredDefinitions.length > 0) ? filteredDefinitions[0].name : '(invalid)';
 };
 
-export const getRiskStatus = (levelOfRisk: number, mitigationStatus: string) => {
-	const statusIcon = {
-		Icon: RISK_LEVELS_ICONS[mitigationStatus] || null,
-		color: RISK_LEVELS_COLOURS[levelOfRisk].color
-	};
-
-	return statusIcon;
-};
+export const getRiskStatus = (levelOfRisk: number, mitigationStatus: string) => ({
+	Icon: RISK_LEVELS_ICONS[mitigationStatus] || null,
+	color: (typeof levelOfRisk === undefined)
```

Comment:
i don't think the levelOfRisk can ever be undefined? @Charence @m1chalkubiak 

If it's not broken before i'm not sure why we're adding this.

Reply:
You are right, for some reason during saving at some moment I getting undefined value for this variable. I will try to debug this and figure out what causes this.

Reply:
I found the cause of the problem, it was about cleaning the state with temporary information about created risk. Before the new active risk was set.